activemodel/lib/active_model/type/value.rb

```diff
@@ -11,6 +11,10 @@ def initialize(precision: nil, limit: nil, scale: nil)
         @limit = limit
       end
 
+      def serializable?(_)
```

Comment:
Should this be part of the public API?

Reply:
I think it should.  I would like it if active model types can tell us whether or not a value can be serialized for that particular type.  We should add some docs, but  I didn't want to go that far unless everyone was OK with adding this to the public API